.gitignore

```diff
@@ -11,4 +11,6 @@ build
 # !.idea/runConfigurations
 
 # Ignore html generated in backend folder
-backend/**/*.html
\ No newline at end of file
+backend/**/*.html
+
+venv
```

Comment:
I added it on purpose because I used different machines for development and it was easier to keep the IDE config in sync. I will remove it from the repo!

Reply:
Solved in this commit be783bb7e442bd3b41e30b61cbc86e6d94ddd7e7